app/src/renderer/components/governance/ModalDeposit.vue

```diff
@@ -50,7 +50,7 @@
 
 <script>
 import { mapGetters } from "vuex"
-import { uatoms, atoms, SMALLEST } from "../../scripts/num.js"
+import num, { uatoms, atoms, SMALLEST } from "../../scripts/num.js"
```

Comment:
do we need to import `num` and the destructured version?

Reply:
we don't I didn't want to do all the refactors here. I volunteer to do the follow up. I think `num.x` is better for search and replace.